### ISSUE TITLE
Support to switch between relative line numbers an regular numbers in insert mode

### ISSUE THREAD

**Issue by kentcdodds** [aioutecism/amVim-for-VSCode#197]
In vim-mode-plus for Atom, I loved this feature. It's basically:

```
autocmd InsertEnter * :set norelativenumber
autocmd InsertLeave * :set relativenumber
```

You get the benefit of regular line numbers in insert mode when you can't jump around anyway, with the relative line numbers when you're in visual/normal mode when you can jump around :)

**Comment by aioutecism** [comment]
Will check if this is possible with current VSCode Extension APIs.

**Comment by Ggpsv** [comment]
@kentcdodds Where did you set that config in vim-mode-plus?

**Comment by kentcdodds** [comment]
I thought it was the default but perhaps I configured it and forgot or maybe it's reading parts of my .vimrc? I'm not sure.

**Comment by aioutecism** [comment]
Changing line numbers style is possible through TextEditorOptions.
https://code.visualstudio.com/docs/extensionAPI/vscode-api#_a-nametexteditoroptionsaspan-classcodeitem-id218texteditoroptionsspan

I'll find time to do this.

**Comment by dagadbm** [comment]
isn't the vscode config: `"editor.lineNumbers": "relative" `enough?

**Comment by kentcdodds** [comment]
Nope. I want to be able to be in relative when in normal/visual mode, but not in insert mode.

**Comment by aioutecism** [comment]
Hi @kentcdodds 

I implemented the feature in #201.
Would you like to review it?

**Comment by aioutecism** [comment]
Thank you for the review @kentcdodds!
This is now supported in 1.27.0.
